deploy/cluster_role.yaml

```diff
@@ -41,4 +47,16 @@ rules:
   resources:
   - '*'
   verbs:
-  - '*'
\ No newline at end of file
+  - '*'
+- apiGroups:
+  - authorization.openshift.io
+  resources:
+  - rolebindings
+  verbs:
+  - '*'
+- apiGroups:
+  - rbac.authorization.k8s.io
+  resources:
+  - '*'
+  verbs:
+  - '*'
```

Comment:
`rbac.authorization.k8s.io` groups contains following api-resources
```
clusterrolebindings                            rbac.authorization.k8s.io      false        ClusterRoleBinding
clusterroles                                   rbac.authorization.k8s.io      false        ClusterRole
rolebindings                                   rbac.authorization.k8s.io      true         RoleBinding
roles                                          rbac.authorization.k8s.io      true         Role
```

I think, we should give only required minimum permission. with the above rules, we are giving permission to `create/delete` any `clusterrole/clusterrolebinding`, which I believe, we can limit

Reply:
I will revisit this, there were some error during e2e run wrt RBAC and I added this to pass e2e.

Reply:
Done https://github.com/codeready-toolchain/member-operator/pull/53/commits/acfd7ab49241f7782a2259da3f2d108cc6afc426

Reply:
@dipak-pawar Done as per your suggestion https://github.com/codeready-toolchain/member-operator/pull/53/commits/51226a31a8f44adbb07b4d40d994a8a3cccc09b1